cocos/ui/UIAbstractCheckButton.cpp

```diff
@@ -53,6 +54,11 @@ _frontCrossDisabledTexType(TextureResType::LOCAL),
 _zoomScale(0.1f),
 _backgroundTextureScaleX(1.0),
 _backgroundTextureScaleY(1.0),
+_backGroundFileName(""),
```

Comment:
why add these files names? I think it is only useful for Editor. Why not persist it in the Editor side?